design/app-bar/toggle.css

```diff
@@ -0,0 +1,14 @@
+@custom-media --screen-desktop (min-width: 728px);
```

Comment:
I quite don't understand why you removed the centralized file for custom media. Now this custom media is repeated in several files...

Reply:
Oh, thank you for pointing this. I have to remove it from here too.

Reply:
Ideally this custom media should leave in a single file. I achieved this in the #28